overlay/files/system/oem/10_accounting.yaml

```diff
@@ -48,3 +48,8 @@ stages:
       commands:
         - chown -R root:admin /usr/local/cloud-config
         - chmod 770 /usr/local/cloud-config
+    - name: "Enable serial login for alpine"
```

Comment:
Nice! Maybe a link to something that documents this? e.g. I found this one: https://access.redhat.com/articles/3166931

Reply:
lets not link to red hat articles as they are usually paywalled :D

Reply:
got a link from alpine wiki!